docs/getting-started/install-from-npm.md

```diff
@@ -34,3 +34,13 @@ Alternatively when only map functionality is needed, and not stock.
 var Highcharts = require('highcharts/highmaps');
 ```
 
+Installing nightly builds of Highcharts
+--------------------------------------
+Nightly builds allows the access to the next Highcharts version prior to release and testing. Currently this is available via the [highcharts-dist nightly branch](https://github.com/highcharts/highcharts-dist/tree/nightly) on Github.
+Note that we do not recommend the use of the nightly build in production environments as these **may contain bugs and are not considered stable.**
+
+The nightly builds can be installed by running `npm install --save highcharts/highcharts-dist#nightly`. 
+
+As this is a dependency to a Github branch you will not be able to update the dependency using `npm update`. 
+Updating requires that you uninstall and then reinstall by running `npm uninstall highcharts && npm install --save highcharts/highcharts-dist#nightly`. 
```

Comment:
I believe `npm install` is enough to update.

Reply:
@bre1470  Are you sure? I didn't find anything regarding this in the npm docs, but I found the following in post which mentions it: https://medium.com/@jonchurch/use-github-branch-as-dependency-in-package-json-5eb609c81f1a

Then I tested locally:

```console
# last npm install was done on friday, copying original node_modules
highcharts-dist-nightly-test $ cp -R node_modules/ node_modules_org
highcharts-dist-nightly-test $ npm i
npm WARN highcharts-nightly-test@1.0.0 No repository field.

audited 1 package in 0.408s
found 0 vulnerabilities

highcharts-dist-nightly-test $ diff -qr node_modules/highcharts/ node_modules_org/highcharts/
highcharts-dist-nightly-test $ (no diff reported)
# No diff found between original and new node_modules found after running npm i

# Testing to remove and reinstall
highcharts-dist-nightly-test $ rm -rf node_modules/highcharts/
highcharts-dist-nightly-test $ npm i
npm WARN highcharts-nightly-test@1.0.0 No repository field.

added 1 package from 1 contributor and audited 1 package in 5.318s
found 0 vulnerabilities

highcharts-dist-nightly-test $ diff -qr node_modules/highcharts/ node_modules_org/highcharts/
Files node_modules/highcharts/es-modules/masters/highcharts-3d.src.js and node_modules_org/highcharts/es-modules/masters/highcharts-3d.src.js differ
Files node_modules/highcharts/es-modules/masters/highcharts-gantt.src.js and node_modules_org/highcharts/es-modules/masters/highcharts-gantt.src.js differ
Files node_modules/highcharts/es-modules/masters/highcharts-more.src.js and node_modules_org/highcharts/es-modules/masters/highcharts-more.src.js differ
...
...
Files node_modules/highcharts/es-modules/masters/highcharts.src.js and node_modules_org/highcharts/es-modules/masters/highcharts.src.js differ

# Diffs are found (i.e the node_modules dir was only updated after deleting it first)
```
Unless I'm missing something obvious this seems that npm doesn't update a github branch dependency unless removed first.

Reply:
Okay, so what I wrote was incorrect. I guess it only works for me, because I switch between local file and github source.